test/functional/tool_utxo_to_sqlite.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test utxo-to-sqlite conversion tool"""
+import os
+try:
+    import sqlite3
```

Comment:
Is the `import` expected to fail on CI?
If so, I am not sure that the test below can run successfully if `sqlite3` is not available...